rdmo/accounts/middleware.py

```diff
@@ -0,0 +1,49 @@
+"""Terms and Conditions Middleware"""
+# ref: https://github.com/cyface/django-termsandconditions/blob/main/termsandconditions/middleware.py
+
+from django.conf import settings
+from django.http import HttpResponseRedirect
+from django.urls import reverse
+
+from .models import ConsentFieldValue
+
+# these exclude url settings are optional
```

Comment:
My philosophy in RDMO was to add all settings to `rdmo.core.settings` in order to avoid those "checking" blocks. (`getattr` is already better than `try ... execpt`.) I suggest we stick to this.